lib/msf/core/db_manager/host.rb

```diff
@@ -9,27 +9,17 @@ def del_host(wspace, address, comm='')
   }
   end
 
+  # Deletes Loot entries based on the IDs passed in.
+  #
+  # @param opts[:ids] [Array] Array containing Integers corresponding to the IDs of the Loot entries to delete.
```

Comment:
The comments mention loot is being deleted, however, this is the `delete_host` method in `db_manager/host`. Should this be host IDs?

Reply:
Copy/paste error. I'm really bad about those lol.